Backend/CacheTags.php

```diff
@@ -95,7 +101,7 @@ public function get255ByteCacheTagHeaderStringValues($cacheTagList) {
      * @param $tags
      */
     public function purgeCacheTags(array $tags) {
-        if(!empty($tags)) {
+        if (!empty($tags) && $this->dataStore->get(\CF\API\Plugin::SETTING_PLUGIN_SPECIFIC_CACHE_TAG) === true) {
```

Comment:
I don't like loose comparisons in PHP, too much potential for bugs.  PHP evaluates a lot of  things we logically consider `false` to `true` ([0.00 for example](http://php.net/manual/en/language.types.boolean.php#74827)).  `===` checks type and value and eliminates all the weird logic the language has to "help" you.